### PR TITLE
Coverpkg support

### PR DESCRIPTION
**Pull request by johnnadratowski**
@onsi - This is for adding support for coverage of external modules to ginkgo.  This simply uses the go test -coverpkg command to add external package coverage.

I've added a couple of tests, similar to the ones you had for normal coverage, to the integration tests.

One minor design detail I should mention is that I updated the checks for cover, so that if -coverpkg is specified without -cover, it will still run coverage.  I think this is in-line with your original design of -cover including -coverprofile.  However, if you want me to change it, I can.

### CODE REVIEW

File: integration/coverage_test.go [hunk 1]
```diff
@@ -1,11 +1,12 @@
 package integration_test
 
 import (
+	"os"
+	"os/exec"
```
Comment (johnnadratowski) [inline integration/coverage_test.go]: This is just from `goimports` shuffling some things around

**Comment by johnnadratowski** [comment]
@onsi - BTW - I don't mind helping update the docs for these PRs if you want the help.

**Comment by onsi** [comment]
Sweet thanks!

Please do update the GH-pages branch when you get a chance. Will merge this in soon :)

**Comment by johnnadratowski** [comment]
@onsi - Any word on this?  I was hoping to get it in soon.  If there is anything else you'd like me to do let me know.

**Comment by onsi** [comment]
woops sorry about that!  pulling it in now.

**Pull request merged**